modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-configuration-resource-overview-{context}"]
+= Image Registry Operator configuration
+
+The image-registry resource offers the following configuration fields.
+
+[cols="3a,8a",options="header"]
+|===
+|Component |Description
+
+|*ManagementState*
+|*Managed:* the operator will update the registry as configuration resources
+are updated.
+*Unmanaged:* the operator will ignore changes to the configuration resources.
+
+|*Removed*
+|The operator will remove the registry instance and tear down any
+storage that the operator provisioned.
+
+|*Logging*
+|Sets `loglevel` of the registry instance.
+
+|*HTTPSecret*
+|Value needed by the registry to secure uploads, generated by default.
+
+|*Proxy*
+|Defines the Proxy to be used when calling master API
+and upstream registries.
+
+|*Storage*
+|*Storagetype:* details for configuring registry storage, for example S3 bucket
+coordinates. Normally configured by default.
+
+|*Requests*
+| API Request Limit details. Controls how many parallel requests a given registry
+instance will handle before queuing additional requests.
+
+|*DefaultRoute*
+|Determines whether or not an external route is defined using the default
+hostname. If enabled, the route uses re-encrypt encryption. Defaults to false.
+
+|*Routes*
+|Array of additional routes to create. You provide the hostname and certificate
+for the route.
+
+|*Replicas*
+|Replica count for the registry.
+
+|===
+
+== Additional configuration resources
```

Comment:
Is there more than just the CA config? If not, I'd break this whole section out into a separate module.